Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -259,10 +259,6 @@ file that was distributed with this source code.
 
         {% if sonata_admin is defined and sonata_admin_enabled %}
             {% set div_class = div_class ~ ' sonata-ba-field-' ~ sonata_admin.edit ~ '-' ~ sonata_admin.inline %}
-
-            {% if sonata_admin.edit == 'inline' and sonata_admin.inline == 'table' %}
```

Comment:
Because adding the ``form-inline`` class was a mistake (it may breaks things), the good fix is the new css one ;)